pycode/memilio-epidata/memilio/epidata_test/test_epidata_get_jh_data.py

```diff
@@ -376,6 +378,44 @@ def test_get_JH_Data_Download_omit_raw(self, mock_loadcsv):
         self.assertEqual(df[(df["CountryRegion"] == 'France') & (df["Date"] == "2020-09-26") & (
                 df['ProvinceState'] == 'Martinique')]["Deaths"].item(), 20)
 
+    def test_get_JH_Data_extract_subframe_between_timeframe(self):
```

Comment:
Either patch load csv or write the test data and call with read data set to true. Actually downloading the data should not happen in the test module.

Reply:
Yes, did this.